common/metrics/fake.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
could you change this file name to noop.go?

Reply:
renamed